tests/fixtures/commands/component-check.yml

```diff
@@ -156,6 +160,8 @@
     - touch: composer.json
     - from: sample-composer.lock
       to: composer.lock
+    - mkdir: /vendor/composer/
```

Comment:
Can we make this in a way that we do not need to create these files for every test ?
When checking for this file, we can ignore if it is not found

Reply:
Hi @joaocsilva it's fixed now, by adding a flag to getJson function whether to throw an exception or not.
  Is this a solution that you have in mind?

Reply:
Yes it works 👍